indexer/packages/postgres/src/db/migrations/migration_files/20240906134410_add_fills_created_at_index.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    CREATE INDEX CONCURRENTLY IF NOT EXISTS "fills_createdat_index" ON "fills" ("createdAt");
+    `);
+}
```

Comment:
Is there an index on subaccountId too on the fills table. Can optimize on group bys and joins

Reply:
Yes that was added in a previous PR (https://github.com/dydxprotocol/v4-chain/pull/2174/files)